src/hooks/useGetColoniesMembersCount.ts

```diff
@@ -0,0 +1,177 @@
+import {
+  type SearchableColonyContributorFilterInput,
+  useGetMembersCountQuery,
+} from '~gql';
+
+/**
+ * Wrapper for useGetMembersCountQuery to return the count as an array
+ *
+ * @param {string[]} colonyAddresses - An array of colony addresses to fetch member counts for
+ * @param {SearchableColonyContributorFilterInput} filterOptions - The filter criteria for member attributes
+ * @returns
+ *   - `colonyMemberCounts` (number[]): The count of members for the specified colonies.
+ *   - `isLoading` (boolean): Indicates whether the data is still being fetched.
+ */
+export const useGetColoniesMembersCount = (
+  colonyAddresses: string[],
+  filterOptions: SearchableColonyContributorFilterInput,
+) => {
+  const {
+    data: contributorsCount,
+    fetchMore,
+    loading: contributorsCountLoading,
+  } = useGetMembersCountQuery({
+    variables: {
+      filter: {
+        or: colonyAddresses.map((colonyAddress) => ({
+          and: [
+            { colonyAddress: { eq: colonyAddress } },
+            {
+              ...filterOptions,
+            },
+          ],
+        })),
```

Comment:
Would it make sense to extract this filter into a constant 
```
const colonyAddressesFilter =  {
        or: colonyAddresses.map((colonyAddress) => ({
          and: [
            { colonyAddress: { eq: colonyAddress } },
            {
              ...filterOptions,
            },
          ],
        }))
     }
```
and then use it here `filter: {...colonyAddressesFilter}` as well as below for the `fetchMore`?